Framework/Crystal/test/SCDCalibratePanels2Test.h

```diff
@@ -194,41 +204,73 @@ class SCDCalibratePanels2Test : public CxxTest::TestSuite {
     // Make a clone of the standard peak workspace
     PeaksWorkspace_sptr pws = m_pws->clone();
 
-    // Adjust bank
-    //
+    // Move one bank to the wrong location
     // NOTE: the common range for dx, dy ,dz is +-5cm
-    double dx = 1.1e-2;
-    double dy = -0.9e-2;
-    double dz = 1.5e-2;
+    const std::string bankname = "bank27";
+    double dx = 1.1e-3;
+    double dy = -0.9e-3;
+    double dz = 1.5e-3;
     // prescribed rotation
     double theta = PI / 3;
     double phi = PI / 8;
     double rvx = sin(theta) * cos(phi);
     double rvy = sin(theta) * sin(phi);
     double rvz = cos(theta);
-    double ang = 1.414; // degrees
+    double ang = 0.02; // degrees
     //
-    adjustComponent(dx, dy, dz, rvx, rvy, rvz, ang, bank_xtop, pws);
+    adjustComponent(dx, dy, dz, rvx, rvy, rvz, ang, bankname, pws);
 
     // Run the calibration
     // NOTE: this should bring the instrument back to engineering position,
     //       which is the solution
     runCalibration(filenamebase.string(), pws, false, false, true);
 
-    // Check if the calibration results
-    // -- get a blank workspace for loading cali results
-    // MatrixWorkspace_sptr ws_raw = m_ws->clone();
-    // -- check
-    // bool sameInstrument =
-    // validateCalibrationResults(m_pws, ws_raw, filenamebase.string());
-    // -- assert
-    // NOTE:
-    //    Due to built-in q vector centering, the asseration will always fail.
-    //    We are commenting it out for now.
-    // TS_ASSERT(sameInstrument);
+    // Apply the calibration results
+    MatrixWorkspace_sptr ws = generateSimulatedWorkspace();
+    const std::string xmlFileName = filenamebase.string() + ".xml";
+    IAlgorithm_sptr lpf_alg = AlgorithmFactory::Instance().create("LoadParameterFile", 1);
+    lpf_alg->initialize();
+    lpf_alg->setLogging(LOGCHILDALG);
+    lpf_alg->setProperty("Workspace", ws);
+    lpf_alg->setProperty("Filename", xmlFileName);
+    lpf_alg->execute();
+
+    // check translation
+    V3D pos_wrng = pws->getInstrument()->getComponentByName(bankname)->getRelativePos();
+    V3D pos_ref = m_pws->getInstrument()->getComponentByName(bankname)->getRelativePos();
```

Comment:
Why the relative position? Do `ws` and `m_pws` have the same reference instrument? Why not the absolute position?

Reply:
The RelativePos here means relative to the parent class, not the same meaning as the one in MoveInstrumentComponent.